Makefile

```diff
@@ -27,15 +27,20 @@ endif
 ##@ Default target
 all: gitops ## Install dependencies and build Gitops binary
 
+
+TEST_TO_RUN=./...
 ##@ Test
-unit-tests: dependencies  ## Run unit tests
+unit-tests: dependencies ## Run unit tests
+	which ginkgo || go install github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
It looks like `which ginkgo` will succeed for folks with Ginkgo v1 installed. Is that a problem? I’m guessing from the fact that you’ve upgraded the library that the v1 CLI doesn’t have the functionality this PR needs; is that right?

Reply:
This is not a problem, however the test output won't be as pretty (the table I want won't be there).

Using the ginkgo2 CLI with ginkgo v1 tests is a problem, however, so if I want to be able to use the v2 CLI, I have to update the library to v2.